BUILD_MESON

```diff
@@ -28,24 +42,17 @@ in Meson you need to prepare "Buidling Dir". simply just type:
 
     meson setup builddir/
 
-This command configures "Building dir" in builddir/.
+This command configures "Building directory" in builddir/.
 You can set desired directory (but we suggest dont use dinit's directory such as build/ & src/ or ... as "Buidling dir").
 If everything goes smoothly this will prepare everything to build dinit, dinitctl, and optionaly the shutdown/reboot/halt utilitys by default.
 
-Everything is ready! you can compile Dinit with changing current directory to builddir/ & run:
+Everything is ready! you can compile Dinit with
 
-     meson compile
+     meson compile -C builddir
 
-Note: on old Mesons, you must use "ninja compile" instead of "meson compile".
+Note: on old Mesons, you need to use "ninja -C builddir" instead of "meson -C builddir".
```

Comment:
... instead of "meson **compile** -C builddir"

Reply:
Can you be specific about which versions of Meson do not support "meson compile -C builddir"?

Reply:
Versions before 0.54. I fix these things.